### ISSUE TITLE
Update flag relationships to use new oclif functionality

### ISSUE THREAD

**Issue by Amndeep7** [mitre/saf#710]
https://github.com/oclif/core/pull/468 this got merged a month ago in release 1.16.0

this'll help with some of the more complex flag interactions (like the mappers that are both file and api based)

we're gonna need to upgrade the version of oclif being used by quite a lot i think

**Comment by aaronlippold** [comment]
I agree - let's look into this as I think it would be a big win and allow us to be more interactive with the users?
